src/components/AccountSelector/AccountSelectButton/AccountSelectorButton.tsx

```diff
@@ -0,0 +1,56 @@
+import { ComponentPropsWithoutRef } from "react";
+
+export interface AccountSelectorProps extends ComponentPropsWithoutRef<"div"> {
+  theme?: string;
+  profileimage: React.ReactNode;
+  accountname: string;
+  svg: React.ReactNode;
+  accountbalance: string;
+  children?: React.ReactNode;
+}
+
+export function AccountSelectorButton(props: AccountSelectorProps) {
+  // Specific classes for the account selector button
+  const accountSelectorClass = "flex flex-row items-center justify-between";
```

Comment:
```suggestion
export function AccountSelectorButton(props: AccountSelectorProps) {
const { theme, profileimage, svg, accountName, accountBlance, children, ...rest } = props;

  // Specific classes for the account selector button
  const accountSelectorClass = "flex flex-row items-center justify-between";
```

After adding this const, replace all `props.` with their corresponding values.